client/src/nv_ingest_client/client/interface.py

```diff
@@ -605,6 +734,17 @@ def vdb_upload(self, **kwargs: Any) -> "Ingestor":
 
         return self
 
+    def save_to_disk(
```

Comment:
If output_directory isn't specified, can we just try creating something with tempdir